src/main/java/com/ericsson/ei/controller/FailedNotificationsControllerImpl.java

```diff
@@ -56,25 +54,26 @@
      * @param subscriptionName
      */
     @Override
-    @ApiOperation(value = "Retrieve missed notifications", response = QueryResponse.class)
-    public ResponseEntity<?> getQueryMissedNotifications(@RequestParam("SubscriptionName") final String subscriptionName) {
+    @ApiOperation(value = "Retrieve failed notifications", response = QueryResponse.class)
+    public ResponseEntity<?> getFailedNotifications(@RequestParam("SubscriptionName") final String subscriptionNames) {
```

Comment:
This param should be @RequestParam(required = true) since in raml it's defined as required.

Reply:
Now it is: @RequestParam(defaultValue = "SubscriptionName", required = true)

Reply:
I don't know about the default value though... Shouldn't it be enough with required without any default value set?

Reply:
Well, use of "defaultValue" is wrong... It should be "value", like @RequestParam(value = "SubscriptionName", required = true)

Reply:
And, "value" need to be provided... otherwise parameter and variable name should be the same.